test/jasmine/tests/axes_test.js

```diff
@@ -4,6 +4,7 @@ var Plots = require('@src/plots/plots');
 var Lib = require('@src/lib');
 var Color = require('@src/components/color');
 var tinycolor = require('tinycolor2');
+var hasWebGLSupport = require('../assets/has_webgl_support');
```

Comment:
I'd like to keep `axes_test.js` for test cases in relation to `src/plots/cartesian/axes.js` only.

@monfera can you move your new test cases to one of the `gl2d_` suite of your choice?